src/main/java/org/openrefine/extensions/commons/importer/FileRecord.java

```diff
@@ -11,4 +12,30 @@ public class FileRecord {
     String mId;
     List<String> relatedCategories;
```

Comment:
It would be worth adding the `final` keyword in front of each of those attributes, to make sure their values are preserved throughout the life cycle of the instance. This will force you to migrate initializations of instances to use the constructor you just defined, instead of assigning each field separately.